server/src/main/java/game/snake/Snake.java

```diff
@@ -32,17 +33,22 @@
     private final LinkedList<FinalSnakeChunk> chunks = new LinkedList<>();
     private final BitWithShortHistory fastHistory = new BitWithShortHistory(false);
     public GrowingSnakeChunk currentChunk;
-    @Getter protected double length;
-    @Getter Vector headPosition;
+    @Getter
+    protected double length;
+    @Getter
+    Vector headPosition;
     double headDirection;
     private char currentChunkId;
-    @Setter private byte skin;
-    @Getter private boolean alive = true;
+    @Setter
+    private byte skin;
+    @Getter
+    private boolean alive = true;
     private char nextChunkId = 0;
     private double targetDirection;
     private boolean userWantsFast = false;
     private double lengthBuffer = 0;
-    @Getter private double width;
+    @Getter
+    private double width;
```

Comment:
Can you undo this change? Not sure why this happens but if I format the code it stays on the same line. I will check if there is a setting for this. Sorry.